src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketCapabilitiesClientImplTest.java

```diff
@@ -0,0 +1,39 @@
+package com.atlassian.bitbucket.jenkins.internal.client;
+
+import com.atlassian.bitbucket.jenkins.internal.client.supply.BitbucketCapabilitiesSupplier;
+import com.atlassian.bitbucket.jenkins.internal.model.AtlassianServerCapabilities;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BitbucketCapabilitiesClientImplTest {
+
+    @Mock
+    private BitbucketCapabilitiesSupplier capabilitiesSupplier;
+    @Mock
+    private AtlassianServerCapabilities newCapabilities, cachedCapabilities;
+    @InjectMocks
+    private BitbucketCapabilitiesClientImpl capabilitiesClient;
+
+    @Test
+    public void testGetServerCapabilitiesNoCache() {
+        Mockito.when(capabilitiesSupplier.get()).thenReturn(newCapabilities);
```

Comment:
super nit (for consistency with other places): static import for Mockito methods like `when()` and `verify()`, etc.